src/engraving/dom/engravingobject.cpp

```diff
@@ -475,6 +475,12 @@ void EngravingObject::undoChangeProperty(Pid id, const PropertyValue& v, Propert
                 toEngravingItem(this)->manageExclusionFromParts(v.toBool());
             }
         }
+    } else if (id == Pid::REPEAT_START || id == Pid::REPEAT_END) {
+        // change also coresponding mmRestMeasure
+        Measure* m = const_cast<Measure*>(toMeasure(this)->coveringMMRestOrThis());
```

Comment:
Handling specific logic (child logic) in parent classes is a bad idea. 
You should override the `undoChangeProperty` method and process this logic where it's necessary.

Reply:
@Eism done!